newIDE/app/src/MainFrame/EditorContainers/StartPage/index.js

```diff
@@ -401,21 +462,25 @@ export const StartPage = React.memo<Props>(
                       ? setPreCreationDialogOpen(true)
                       : onCreateFromExampleShortHeader(
                           setIsOpening,
-                          onOpenExample
+                          onOpenProjectJustCreated(true)
                         )(i18n, selectedExample);
                   }}
                 />
               )}
-              {selectedExample && preCreationDialogOpen && (
+              {preCreationDialogOpen && (
                 <LocalProjectPreCreationDialog
                   open
                   onClose={() => setPreCreationDialogOpen(false)}
                   onCreate={() =>
-                    onCreateFromExampleShortHeader(setIsOpening, onOpenExample)(
-                      i18n,
-                      selectedExample,
-                      outputPath
-                    )
+                    selectedExample
+                      ? onCreateFromExampleShortHeader(
+                          setIsOpening,
+                          onOpenProjectJustCreated(true)
+                        )(i18n, selectedExample, outputPath)
+                      : onCreateBlank(onOpenProjectJustCreated(false))(
+                          i18n,
+                          outputPath
```

Comment:
Could we investigate using the promise rather than a callback?

`onCreateBlank(i18n, outputPath).then(onOpenProjectJustCreated(true))`

or even 
```
await onCreateBlank(i18n, outputPath);
onOpenProjectJustCreated(true);
```

looks more readable to me and I think the callback is not providing more value than a promise that would return an object (rather than passing arguments to the callback)?